### PR TITLE
update README.md: Improvement in the steps of Quickstart

### LINKED ISSUE #88: Improverment in the steps of Quickstart
### What would you like to share?

Steps for getting started with the project are not clear from a beginner's perspective, some more information like git commands can be added to make it beginner friendly.

### Would you like to work on this issue?

- [X] Yes, I want to work on this issue!

### PR DESCRIPTION
**Pull request by yashikabhargava**
## Fixes Issue
Closes #88 

## Changes proposed
added git commands to make the steps beginner friendly.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EventCalendar/pull/89"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,24 @@ Here is an example file...
 
 You can use the **Gitpod** badge or to do it manually...
 
-1. Fork this repo
+1. Fork [**this**](https://github.com/EddieHubCommunity/EventCalendar) repository
```
Comment (EmmaDawsonDev) [inline README.md]: I'm not sure of the purpose of a link here when you are already on this page.

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Looks good, just one small change to make please.

File: README.md (hunk 1 above)
Comment (yashikabhargava) [reply README.md]: Yes that's right, I'll remove it.

**Comment by yashikabhargava** [comment]
@emmalearnscode I've done the changes, could you please review it.

**Review by EmmaDawsonDev — APPROVED** [review]
Looks great, thanks for contributing!

**Pull request merged**